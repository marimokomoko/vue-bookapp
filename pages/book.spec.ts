import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Book from '@/pages/book.vue'
let wrapper: Wrapper<Vue>;

describe('画面レイアウト', () => {
    it('スナップショットテスト', () => {
        wrapper = mount(Book, {
            localVue: createLocalVue(),
            vuetify: new Vuetify(),
            stubs: ['router-link']
        })
        expect(wrapper.html()).toMatchSnapshot()
    })
})