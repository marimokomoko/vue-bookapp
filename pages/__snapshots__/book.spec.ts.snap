// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<div>
  <nuxtchild books=\\"\\"></nuxtchild>
</div>"
`;
